deploy/cephcsi/image/Dockerfile

```diff
@@ -33,6 +33,8 @@ RUN dnf -y install \
 	/usr/bin/cc \
 	make \
 	git \
+	&& dnf clean all \
+	&& rm -rf /var/cache/yum \
```

Comment:
This is just a temporary image, only used for compiling the executable. This cleanup does not have any effect on the resulting image. In fact, it will just slow things down a little...

Reply:
considering we do have only one layer ( as its in same RUN command) with reduced size build layer in the image,  I expect there is a small effect. why you say no effect in this context and it can slow things down?

Reply:
The only advantage to add the extra commands to clean the DNF caches, is that (potentially) cached layers on a build machine can be re-used. It could save some storage (as it uses the cached layer) on a developers machine, but automated build environments will rebuild all layers from the same Dockerfile anyway.

So, is the time that `dnf clean all` needs to cleanup worth the few (?) MBs of storage in a developers environment?

Reply:
I think so.. more or less, keeping a cache inside a container layer is not a practice in general.